### DISCUSSION TITLE
[@types/ol] seems no more compatible with Openlayers 6.15.0

### DISCUSSION THREAD

**Discussion by lilas78** [DefinitelyTyped/DefinitelyTyped#63868] [Issues with a @types package] [answered]
With Openlayers 6.15.0, some enum have disappeared (https://github.com/openlayers/openlayers/releases/tag/v6.15.0).

```
`This change only affects users that were using the non-API string enums

ol/OverlayPositioning
ol/extent/Corner
ol/format/FormatType
ol/geom/GeometryType
ol/source/State
ol/source/WMSServerType
ol/source/WMTSRequestEncoding
Instead of these, use the respective strings, which are now typesafe by means of union types`
```

Last version of @types/ol is not compatible with this version

[a comment by an automated account is not shown]

**Comment by hanreev** [comment, marked as answer]
@lilas78
As of v6.6.0, OpenLayers already has built-in TypeScript definitions, you don't need to install @types/ol anymore.

If you need @types/ol compatible definitions, you can use [@hanreev/types-ol](https://github.com/hanreev/types-ol).

**Reply by lilas78** [reply]
Thank you for your answer :-)

I tried to use built-in TypeScript definitions from openlayers. But in this case, version 6.15.1 from openlayers  causes compilation errors with geotiff and geojson modules. It seems there are some dependencies problems. 
It works with [@hanreev/types-ol](https://github.com/hanreev/types-ol) module
